kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -113,17 +126,21 @@ enum State {
         /** The outbound connection is active */
         OUTBOUND_ACTIVE,
         /** The connection to the outbound cluster failed */
-        FAILED
+        FAILED,
```

Comment:
reorder so FAILED comes last.

Reply:
I didn't because I didn't want to change the ordinals, but thinking about that its all runtime state and never persisted so re-numbering is fine. I'll update.